File: Programmers/Level1/이상한문자만들기/이상한문자만들기.py
def solution(s):
    answer = []
    words = s.split(' ')
    for word in words:
        temp = ''
        for i in range(len(word)):
            if i % 2:
                temp += word[i].lower()
            else:
                temp += word[i].upper()
        answer.append(temp)
    return ' '.join(answer)

s = "try hello world"
print(solution(s))

File: Programmers/Level1/이상한문자만들기/이상한문자만들기.js
function solution(s) {
  const wordList = s.split(" ").map((word) => {
    return word
      .split("")
      .map((chr, idx) => {
        return idx % 2 ? chr.toLowerCase() : chr.toUpperCase();
      })
      .join("");
  });

  return wordList.join(" ");
}

const s = "try hello world";

console.log(solution(s));
